vxquery-rest/src/test/resources/vxquery.properties

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
@prestoncarman Yes. My idea is also to use this along with vxquery-server CLI. When starting the vxquery-server, users can specify a properties file by adding **-config foo/bar.properties** to the CLI app arguments (after a **--**. ex: "-- -restPort 8080 -config foo/bar.properties").